platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/wiring/PlatformWiring.java

```diff
@@ -640,6 +652,7 @@ public void bind(
             @NonNull final SavedStateController savedStateController,
             @NonNull final SignedStateHasher signedStateHasher,
             @NonNull final AppNotifier notifier,
+            @NonNull final StateGarbageCollector stateGarbageCollector,
```

Comment:
missing doc

Reply:
I have altered the deal. Pray that I do not alter it further.

(Translation: I've changed the pattern so that this javadoc is no longer neccessary. 😜 )